### PR TITLE
moved seller-listing seperate component for showing listing

### PR DESCRIPTION
**Pull request by vikas-cis**

### CODE REVIEW

File: src/app/market/sell/seller-listing/seller-listing.component.ts [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+  getStatus(status: string) {
+    return [this.status.get(status)];
+  }
+
+  confirmDeleteListing(template: Template): void {
+    const dialogRef = this.dialog.open(DeleteListingComponent);
+    dialogRef.componentInstance.templateToRemove = template;
+    dialogRef.afterClosed().subscribe(
+      () => this.callLoad.emit()
```
Comment (kewde) [inline src/app/market/sell/seller-listing/seller-listing.component.ts]: Can you perhaps get element reference to the SellerListingComponent element and delete it?
That way we don't have to reload the page.

**Comment by kewde** [comment]
Please fix tests.

**Comment by vikas-cis** [comment]
@kewde seems like need to restart :P

File: src/app/market/sell/sell.component.ts [hunk 2]
```diff
@@ -182,4 +182,9 @@ export class SellComponent implements OnInit {
     return this.pages[0].pageNumber;
   }
 
+  // Delete the listing using index
+  deleteListing(listIndex: number, index: number) {
```
Comment (kewde) [inline src/app/market/sell/sell.component.ts]: Use pageIndex, and listingIndex as the keywords
Also fix the order, page goes before listing

**Pull request merged**